CityPuzzle/Classes/Sql.cs

```diff
@@ -267,6 +268,31 @@ private static List<Lazy<Puzzle>> ConvertTasks(string strtask)
                 Quest = puzzle.Value.Quest};
             return p;
         }
-       
+
+        public static SimpleUser GetCurrentUser()
+        {
+            List<SimpleUser> info;
+            using (SQLiteConnection conn = new SQLiteConnection(App.FilePath))
+            {
+                conn.CreateTable<SimpleUser>();
+                info = conn.Table<SimpleUser>().ToList();
+            }
+            return info[0];
+        }
+
+        public static void SaveCurrentUser(User user)
+        {
+            SimpleUser simpleUser = new SimpleUser(user);
+            using (SQLiteConnection conn = new SQLiteConnection(App.FilePath))
+            {
+                user.QuestsCompleted = null;
```

Comment:
Sql cant save user with null questsCompleted variable.

Reply:
My table is simpleUser format and I am using SQLiteConnection (not Sql), because SQLiteConnection does not allow saving generic list.

Reply:
And this is the reason why I use simpleUser.

Reply:
Okay then